cmd/devices_list.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2021 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/mendersoftware/mender-cli/client/devices"
+)
+
+// const (
+// 	argDetailLevel = "detail"
+// )
+
+var devicesListCmd = &cobra.Command{
+	Use:   "list",
+	Short: "List Mender Devices from the Mender server.",
+	Run: func(c *cobra.Command, args []string) {
+		cmd, err := NewDevicesListCmd(c, args)
+		CheckErr(err)
+
+		CheckErr(cmd.Run())
+	},
+}
+
+func init() {
+	devicesListCmd.Flags().IntP(argDetailLevel, "d", 0, "devices list detail level [0..3]")
+}
+
+type DevicesListCmd struct {
+	server      string
+	skipVerify  bool
+	tokenPath   string
+	detailLevel int
```

Comment:
minor/maybe stupid: can we name 0 1 2 3 detail levels? as in: Normal Verbose ...

Reply:
Same here: I'll leave it for a further rework (together with artifacts list)